src/backend/OpenWeatherMap.js

```diff
@@ -91,7 +91,7 @@ module.exports = class OpenWeatherMap extends require('./Backend') {
         sunrise: data.current.sunrise,
         sunset: data.current.sunset,
         temp: new Temperature(data.current.temp, unit),
-        time: data.current.dt,
+        time: data.current.dt * 1000,
```

Comment:
apparently OWM was using second ts, whereas everywhere else in our system we use ms ts